Cargo-zng.toml

```diff
@@ -38,5 +38,14 @@ libc = "0.2.43"
 [build-dependencies]
 cmake = "0.1.50"
 
+[features]
+# Experimental: This feature only affects zlib-ng, is only relevant to riscv64
+# targets, and currently only has an effect when `cmake` is used. It builds
+# binaries portable to machines without RVV, even if the build machine is
+# riscv64 and detected to support RVV. This is useful for building a more
+# portable binary, or on systems where auto-detection is incorrect
+# (as in https://github.com/zlib-ng/zlib-ng/issues/1705).
```

Comment:
I think here one would also have to advertise the `RVV_OFF` environment variable.

Reply:
I've added [d0d8002](https://github.com/rust-lang/libz-sys/pull/218/commits/d0d8002bedff3898ec6de6a269c3afb61a49fb97), which expands the comments to also document the `RVV_OFF` environment variable.